avocado/utils/pci.py

```diff
@@ -141,8 +141,7 @@ def get_slot_from_sysfs(full_pci_address):
     if not os.path.isfile("/proc/device-tree/%s/ibm,loc-code" % devspec):
         return
     slot = read_file("/proc/device-tree/%s/ibm,loc-code" % devspec)
-    return re.match(r'((\w+)[\.])+(\w+)-(\w*\d+)-(\w*\d+)|Slot(\d+)',
-                    slot).group()
```

Comment:
Does this means that the format is arbitrary and there is no suitable regexp, so you want the users to match it themselves? I found that in qemu they use `qemu_<name>:<phb-index>:<slot>.<fn>` for their devices and the physical host's for passthrough devices. So I think it should be possible to split it by `:` and the last element by `.` to get the components. Anyway I'm not using this code, I'd just like to understand the change and the commit message did not help much.